packages/adapter-cloudflare-workers/index.js

```diff
@@ -149,29 +149,34 @@ export default function ({ config = 'wrangler.toml', platformProxy = {} } = {})
 			builder.writePrerendered(bucket_dir);
 		},
 
-		async emulate() {
-			const proxy = await getPlatformProxy(platformProxy);
-			const platform = /** @type {App.Platform} */ ({
-				env: proxy.env,
-				context: proxy.ctx,
-				caches: proxy.caches,
-				cf: proxy.cf
-			});
+		emulate() {
+			// we want to invoke `getPlatformProxy` only once and await it only when it is accessed
```

Comment:
```suggestion
			// we want to invoke `getPlatformProxy` only once, but await it only when it is accessed.
			// If we would await it here, it would hang indefinetly because the platform proxy only resolves once a request happens
```